discovery-provider/src/tasks/entity_manager/entities/track.py

```diff
@@ -285,12 +269,16 @@ def get_handle(params: ManageEntityParameters):
         .first()
     )
     if not handle or not handle[0]:
-        raise IndexingValidationError(f"Cannot find handle for user ID {params.user_id}")
+        raise IndexingValidationError(
+            f"Cannot find handle for user ID {params.user_id}"
+        )
 
     return handle[0]
 
 
-def update_track_record(params: ManageEntityParameters, track: Track, metadata: Dict, handle: str):
+def update_track_record(
+    params: ManageEntityParameters, track: Track, metadata: Dict, handle: str
+):
     populate_track_record_metadata(track, metadata, handle)
     track.metadata_multihash = params.metadata_cid
```

Comment:
`track.metadata_multihash` should be the manually computed cid of the entire metadata blob with updates applied, not `params.metadata_cid`, no?

Reply:
Oh yup totally, working on a fix now! I think it will be better to structure all of them like `user` for consistency anyway

Reply:
After chatting with @isaacsolo we decided to remove saving of the metadata blobs for tracks/playlists altogether because those blobs aren't referenced anywhere. That means once we backfill collectibles & associated wallets we can drop cid_data and only have to remove this logic for users

Reply:
sounds good!